<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelAdmin</class>
 <widget class="QDialog" name="PanelAdmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Gestion panels</string>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2" rowspan="4">
    <layout class="QVBoxLayout" name="switch_container">
     <item>
      <spacer name="top_spacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="move_right">
       <property name="text">
        <string notr="true">&gt;&gt;&gt;&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="move_left">
       <property name="text">
        <string notr="true">&lt;&lt;&lt;&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="bottom_spacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QListWidget" name="panels">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="products_management_button">
     <property name="text">
      <string notr="true">Gerer les produits</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="5">
    <widget class="GlobalConsumptionList" name="product_list">
     <property name="showDropIndicator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::NoDragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::IgnoreAction</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="animated">
      <bool>false</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>false</bool>
     </property>
     <attribute name="headerShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">Consommations</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="Input" name="name_input">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="3" rowspan="5">
    <widget class="PanelConsumptionList" name="panel_content">
     <property name="showDropIndicator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::NoDragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::IgnoreAction</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string notr="true">Consomations affichés pour le panel</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="hidden">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string notr="true">Panel caché</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QWidget" name="button_container" native="true">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="add_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string notr="true">Ajouter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delete_button">
        <property name="text">
         <string notr="true">Supprimer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Input</class>
   <extends>QLineEdit</extends>
   <header>gui/input_widget</header>
  </customwidget>
  <customwidget>
   <class>PanelConsumptionList</class>
   <extends>QTreeWidget</extends>
   <header>gui/panels_management_window</header>
  </customwidget>
  <customwidget>
   <class>GlobalConsumptionList</class>
   <extends>QTreeWidget</extends>
   <header>gui/panels_management_window</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>add_button</sender>
   <signal>clicked()</signal>
   <receiver>PanelAdmin</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delete_button</sender>
   <signal>clicked()</signal>
   <receiver>PanelAdmin</receiver>
   <slot>delete()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>panels</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>PanelAdmin</receiver>
   <slot>on_selection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>move_right</sender>
   <signal>clicked()</signal>
   <receiver>PanelAdmin</receiver>
   <slot>add_product()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>504</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>move_left</sender>
   <signal>clicked()</signal>
   <receiver>PanelAdmin</receiver>
   <slot>delete_product()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>523</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>518</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hidden</sender>
   <signal>toggled(bool)</signal>
   <receiver>PanelAdmin</receiver>
   <slot>on_hide(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>products_management_button</sender>
   <signal>clicked()</signal>
   <receiver>PanelAdmin</receiver>
   <slot>consumption_management_fnc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>503</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>delete()</slot>
  <slot>on_selection()</slot>
  <slot>delete_product()</slot>
  <slot>add_product()</slot>
  <slot>on_hide(bool)</slot>
  <slot>consumption_management_fnc()</slot>
 </slots>
</ui>
